packages/curve-ui-kit/src/shared/ui/Button/Button.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button as MuiButton, ButtonProps as MuiButtonProps } from '@mui/material'
+
+export interface ButtonProps extends MuiButtonProps {
+  label: string
```

Comment:
Using children is much cleaner and allows for more flexibility. E.g. using icons is impossible if you require string.
```suggestion
```